### PR TITLE
Change publisher from "PeterJausovec" to "ms-azuretools"

### PR DESCRIPTION
**Pull request by EricJizbaMSFT**

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Azure Docker API (Preview)
 
-The following extension commands are supported for programmatic use. If a parameter is not specified, the user will be prompted for the value. You must list 'PeterJausovec.vscode-docker' under the 'extensionDependencies' section of your package.json to ensure these apis are available to your extension.  (**Note: the publisher will soon be changing, and although we will attempt to keep this change from affecting dependents, it would be good to contact us if you are using this API so we can inform you of the change.**)
+The following extension commands are supported for programmatic use. If a parameter is not specified, the user will be prompted for the value. You must list 'ms-azuretools.vscode-docker' under the 'extensionDependencies' section of your package.json to ensure these apis are available to your extension.  (**Note: the publisher will soon be changing, and although we will attempt to keep this change from affecting dependents, it would be good to contact us if you are using this API so we can inform you of the change.**)
```
Comment (PrashanthCorp) [inline docs/api.md]: "Note: The publisher _has changed_" ?

**Comment by rcjsuen** [comment]
I won't forget you, Peter Jausovec!

**Comment by StephenWeatherford** [comment]
I'm seeing lots of code matches to PeterJausovec.vscode-docker in GitHub.

**Comment by EricJizbaMSFT** [comment]
@StephenWeatherford the marketplace is already updated so merging this to get a build ASAP: https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker

> I'm seeing lots of code matches to PeterJausovec.vscode-docker in GitHub.

There's already an offline discussion going on that you're a part of. Following up there.

**Comment by peterj** [comment]
noooo :D 👍

**Pull request merged**